src/web/layouts/DecideLayout.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { designTypeDefault } from '@root/src/lib/designTypes';
 
-import { StandardLayout } from './StandardLayout';
-import { ShowcaseLayout } from './ShowcaseLayout';
+import { StandardLayout } from './StandardLayout/StandardLayout';
```

Comment:
I think my preference would be index files in this case (I think I overheard you mention this?)

Reply:
I normally would use index for cases like these, I think its something we should agree on as a team, I suspect @oliverlloyd prefers not to have multiple index file names, I dont mind either way

Reply:
I can see the flip side the files being less searchable (because you're then mixing up paths and files). Could use something like https://www.npmjs.com/package/directory-named-webpack-plugin but I'm loathed to increase webpack config complexity too 😆

Reply:
The problem I have with index files is it makes navigating the codebase harder.

`import {MyThing} from '../MyThing' // using index.js` is concise but depends on convention to prevent multiple exports or renaming

`import {MyThing} from '../MyThing/MyThing'` is verbose but explicit

🤷‍♂ I'm not super fussed though to be honest. If the consensus (eg, you both want to) is to use them that's fine. Just so long as we don't do:

```
// index.js
export {MyThing as AnotherThing} from './MyThing';
export {RandomThing} from './RandomThing'; // Directory spaghetti begins here
```

Reply:
yeah verbose but explicit is generally my go to stance.  👍

Reply:
I will leave it as it in the meantime, and we can maybe discuss this as a team in standup and do a quick vote?